codebleu/test_codebleu.py

```diff
@@ -0,0 +1,180 @@
+import json
+import unittest
+import collections
+import codebleu_metric as cbl
+import codebleu.graph_generator.graphgenerator as gg
+import networkx as nx
+from graph_generator.graphgenerator import AstGraphGenerator
+from graph_generator.graphgenutils import prettyprint_graph
+from graph_generator.type_lattice_generator import TypeLatticeGenerator
+from tree_sitter import Language, Parser
+
+
+class TestCodebleuMetric(unittest.TestCase):
```

Comment:
Let's remove the inheritance from unittest and use pytest instead. Inhereting from `unittest.TestCase` prevents form using fixtures, parameterized tests, etc.
```suggestion
class TestCodebleuMetric:
```